pkg/client/unversioned/request.go

```diff
@@ -570,6 +570,9 @@ func (r *Request) Watch() (watch.Interface, error) {
 		client = http.DefaultClient
 	}
 	resp, err := client.Do(req)
+	//When the apiserver goes down, we can't log response codes.
```

Comment:
Space after //

Did you mean to say "Log response codes even if there was an error"?